internal/payloads/validation/id_test.go

```diff
@@ -0,0 +1,59 @@
+package validation
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestDigitsOnlySucceeds(t *testing.T) {
+	id1 := "1111"
+	err := DigitsOnly(id1)
+	require.NoError(t, err)
+
+	id2 := "1"
+	err = DigitsOnly(id2)
+	require.NoError(t, err)
+
+	id3 := "0"
+	err = DigitsOnly(id3)
+	require.NoError(t, err)
+}
+
+func TestDigitsOnlyWithSymbolsFails(t *testing.T) {
+	id1 := "-1111"
+	err := DigitsOnly(id1)
+	if err == nil {
+		t.Errorf("no error for invalid input: %s", id1)
+	}
```

Comment:
As I said, negative primary key is a valid key. It is rather theoretical because we will never run out of positive numbers this century but still.

Reply:
Oh, I see. Okay, I am sorry. I misunderstood your previous comment on this issue. :+1: